tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "target": "es2015",
+        "target": "es2019",
```

Comment:
Why move to es2019?  No objection, just curious?  (Though it probably means we need a major version bump anyway, just for this.)

Reply:
I was using `Object.entries()` and `Object.fromEntries()` in the tests, but I've eliminated that now with the move to not supporting WP4